events/v1/views.py

```diff
@@ -13,35 +12,6 @@
 from events.v1.utils import get_similar_events
 
 
-class EventTypeListView(APIView):
-    """ View for listing the event types """
-
-    @extend_schema(
-        responses={
-            200: {
-                "type": "array",
-                "items": {
-                    "type": "object",
-                    "properties": {
-                        "label": {"type": "string", "example": "SESSION"},
-                        "key": {"type": "string", "example": "session"}
-                    }
-                },
-                "example": [
-                    {"label": "SESSION", "key": "session"}
-                ]
-            }
-        }
-    )
-    def get(self, request, *args, **kwargs):
-        """ Get the event types """
-        event_types = [
-            {"label": choice[0], "key": choice[1]}
-            for choice in Event.EventType.choices
```

Comment:
@arslanather do we need to remove `event_type` field from Event model as well?

Reply:
No, it was basically not necessary to send event types to frontend since they are constant and we can maintain a list on frontend

Reply:
Sounds good!